src/mococrw/padding_mode.h

```diff
@@ -77,9 +78,9 @@ class NoPadding: public RSAPadding {
     * @param key RSA public key that will be used for encryption
     * @return the maximum size of the data that can be encrypted in bytes.
     */
-    int getDataMaxSize(const AsymmetricPublicKey& key) const override
+    int getDataMaxSize(AsymmetricPublicKey& key) const override
     {
-        return openssl::_RSA_size(key.internal()->pkey.rsa);
+        return key.getKeySize()/8;
```

Comment:
Maybe a general block size function? It is unclear if the data block size always is exactly the key size?